db/migrate/20200217181820_add_application_authentications.rb

```diff
@@ -0,0 +1,11 @@
+class AddApplicationAuthentications < ActiveRecord::Migration[5.2]
+  def change
+    create_table :application_authentications do |t|
+      t.references :tenant,         :index => true, :null => false, :foreign_key => {:on_delete => :cascade}
+      t.references :application,    :index => true, :null => false, :foreign_key => {:on_delete => :cascade}
```

Comment:
Will this delete the `application` if the `application_authentication` is deleted?

Reply:
No it will delete this record if the application is deleted